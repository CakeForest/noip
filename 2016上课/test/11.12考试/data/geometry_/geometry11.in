100
212 628 300 248 295 541 925 160 233 632 776 288 302 400 588 282 870 920 920 832 66 266 815 96 864 820 384 458 183 852 474 865 440 272 500 972 46 544 74 128 728 665 274 622 500 963 89 795 495 372 808 296 503 468 558 747 716 128 345 450 552 824 739 84 790 79 825 860 240 401 240 676 29 520 42 956 828 269 156 924 344 640 106 45 10 776 952 235 24 560 16 920 132 916 522 276 163 718 876 160 
600 367 800 832 706 188 862 609 827 712 515 612 296 962 508 844 950 20 888 354 410 616 829 352 959 88 660 805 10 464 786 36 392 655 455 454 0 645 608 254 622 40 663 650 712 686 475 315 338 309 95 160 968 762 25 905 136 868 11 400 656 600 994 6 690 14 240 700 222 156 943 60 600 396 184 11 0 368 988 645 484 120 164 560 618 557 405 420 448 815 725 640 355 998 600 690 425 838 450 370 
100
601 84
560 645
986 640
80 219
605 970
514 453
438 634
341 520
74 536
680 408
416 793
374 123
463 624
35 520
386 870
792 836
726 480
877 807
184 18
239 416
748 40
878 376
264 82
932 900
215 676
554 33
513 94
520 915
540 230
772 776
678 417
560 740
319 828
642 336
239 676
541 932
340 340
990 712
449 40
880 631
994 536
598 460
340 339
378 631
528 400
424 477
886 336
400 480
326 491
846 680
8 953
499 931
514 160
25 460
5 272
850 920
348 950
156 546
728 592
288 980
856 402
538 970
522 114
160 440
260 404
50 704
860 250
267 460
810 570
224 600
836 228
674 136
635 472
376 507
900 456
380 864
455 29
190 227
200 432
560 445
65 160
425 872
112 678
755 965
995 736
760 680
91 505
890 246
128 28
772 725
620 384
52 792
4 224
464 538
865 176
889 360
210 418
916 96
794 990
144 185
