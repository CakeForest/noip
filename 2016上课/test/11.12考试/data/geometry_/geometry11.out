68
100
100
27
100
97
100
88
60
100
100
50
100
51
100
100
100
100
22
66
76
100
39
100
90
61
62
100
77
100
100
100
100
99
92
100
68
100
50
100
100
100
68
100
92
91
100
90
82
100
95
100
68
48
24
100
100
73
100
100
100
100
65
60
68
78
100
74
100
83
100
79
100
90
100
100
50
42
62
100
22
100
79
100
100
100
56
100
17
100
100
87
22
100
100
100
62
100
100
34
